tests/run/cpython_capi.pyx

```diff
@@ -5,18 +5,36 @@ from cpython cimport mem
 from cpython.pystate cimport PyGILState_Ensure, PyGILState_Release, PyGILState_STATE
 
 
+cdef short _assert_calloc(short * s, int n):
```

Comment:
In order to be able to call this function from a `nogil` section _and_ raise exceptions from it, it needs to be declared:
```cython
cdef short _assert_calloc(short* s, int n) except -1 with gil:
```

Reply:
Fixed it myself, since I was at it anyway.